parts/mini-cart.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:woocommerce/mini-cart-contents -->
+<div class="wp-block-woocommerce-mini-cart-contents"><!-- wp:woocommerce/filled-mini-cart-contents-block -->
+<div class="wp-block-woocommerce-filled-mini-cart-contents-block"><!-- wp:woocommerce/mini-cart-title-block -->
+<div class="wp-block-woocommerce-mini-cart-title-block"></div>
+<!-- /wp:woocommerce/mini-cart-title-block -->
+
+<!-- wp:woocommerce/mini-cart-items-block -->
+<div class="wp-block-woocommerce-mini-cart-items-block"><!-- wp:woocommerce/mini-cart-products-table-block -->
+<div class="wp-block-woocommerce-mini-cart-products-table-block"></div>
+<!-- /wp:woocommerce/mini-cart-products-table-block --></div>
+<!-- /wp:woocommerce/mini-cart-items-block -->
+
+<!-- wp:woocommerce/mini-cart-footer-block -->
+<div class="wp-block-woocommerce-mini-cart-footer-block"></div>
+<!-- /wp:woocommerce/mini-cart-footer-block --></div>
+<!-- /wp:woocommerce/filled-mini-cart-contents-block -->
+
+<!-- wp:woocommerce/empty-mini-cart-contents-block -->
+<div class="wp-block-woocommerce-empty-mini-cart-contents-block"><!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center"><strong>Your cart is currently empty!</strong></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:woocommerce/mini-cart-shopping-button-block -->
+<div class="wp-block-woocommerce-mini-cart-shopping-button-block"></div>
+<!-- /wp:woocommerce/mini-cart-shopping-button-block --></div>
+<!-- /wp:woocommerce/empty-mini-cart-contents-block --></div>
+<!-- /wp:woocommerce/mini-cart-contents -->
```

Comment:
Oh, I actually was going to ask about all the Woo integrations we did with Go. This looks like a way where we are still providing some Woo blocks to users in this `mini-cart` area. Anything else I should know about the Woo integrations?

Reply:
We still have some testing to do with Woo but we did a lot of customizations specific to Go that are being ripped out. We will need to do a sweep of styles in the future but the flyout cart was something custom we had that WooCommerce now provides a block for.